src/EventListener/UsageStatisticsListener.php

```diff
@@ -74,7 +74,7 @@ protected function logUsageStatistics(Request $request): void
 
         $params = $this->getParams($request);
         $user = $this->userResolver->getUser();
-        $this->logger->info($request->attributes->get('_controller'), [
+        $this->logger->info($request->attributes->get('_controller') ?? '', [
```

Comment:
I would prefer the default value parameter
```suggestion
        $this->logger->info($request->attributes->get('_controller', ''), [
```